app/controllers/webhook_deliveries_controller.rb

```diff
@@ -0,0 +1,29 @@
+class WebhookDeliveriesController < ApplicationController
+  respond_to :html, :js
+
+  before_action :set_namespace
+  before_action :set_webhook
+  before_action :set_webhook_delivery
```

Comment:
Whenever that happens, it will be trivial to separate it into a separate action :wink: